integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -533,8 +538,26 @@ def export(self, **kwargs):
                         existing_record.channel_metadata = transformed_item
                         update_payload[key] = existing_record
 
-            for key, item in items_to_delete.items():
-                delete_payload[key] = item
+            if items_to_delete:
```

Comment:
If we want to sanitize the item to be deleted we have to now request the metadata for it.